packages/runtime/src/state/react-builder-preview.ts

```diff
@@ -500,9 +501,11 @@ export function messageChannelMiddleware(): Middleware<Dispatch, State, Dispatch
       const breakpoints = ReactPage.getBreakpoints(state)
       messageChannel.port1.postMessage(setBreakpoints(breakpoints))
 
-      const locales = ReactPage.getLocales(state).locales
-      const defaultLocale = ReactPage.getLocales(state).defaultLocale
-      const routerLocale = Router.locale
+      const locales = ReactPage.getLocales(state).map(locale =>
+        localeStringSchema.parse(locale.toString()),
+      )
+      const defaultLocale = localeStringSchema.parse(ReactPage.getDefaultLocale(state)?.toString())
```

Comment:
Is the idea that this will just throw an error if `getDefaultLocale` returns a nullish value?

Reply:
Yeah, that was bad. Updated so we parse in the action creator instead.